Services/Simulations.cs

```diff
@@ -367,6 +379,82 @@ public async Task DeleteAsync(string id)
             await this.simulationsStorage.DeleteAsync(id);
         }
 
+        public async Task TrySeedAsync(string templateName)
```

Comment:
can we move all this new code into Services/Simulations, and from Startup call that?  Having the code here seems quite strange because the agent doesn't use it. I think that Startup making a service call makes it easier to understand what happens

Reply:
done